pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -133,17 +155,79 @@ func (r *ReconcileComputeNodeOpenStack) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	// ScriptsConfigMap
```

Comment:
All these (lines 158-220) is simply to get the openstack admin secret and the client, right? perhaps worth to move it to a function to help keeping the reconcile loop as simple as possible

Reply:
no, L158-L196 is to get, create or update the configmap which holds the PRE/POST scripts. L198-L220 id for the admin secret and configmap

Reply:
moved into 2 functions

Reply:
awesome